afs/plugins/init-workflow.json

```diff
@@ -15,6 +15,15 @@
                 "circleColor": "#b0adf3",
                 "desc": "A research effort, study, or conservation project"
             },
+            {
+                "workflowid": "ef3a374c-f795-4317-aaee-83e1493eabb6",
+                "slug": "project-collection-workflow",
+                "name": "Update Project Collection",
```

Comment:
Could the name be changed to "Manage Objects in a Project"?

Reply:
updated